protocols/118e8c/README.md

```diff
@@ -0,0 +1,33 @@
+# FluoGene HLA NX 96-Well Setup
+
+### Author
+[Opentrons](https://opentrons.com/)
+
+## Categories
+* qPCR Setup
```

Comment:
I would like to try to prevent additional top-level categories unless we collectively feel the need for a new one. Can this be changed to `PCR`?